app/src/main/java/com/my/playlistmaker/TrackAdapter.kt

```diff
@@ -0,0 +1,56 @@
+package com.my.playlistmaker
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+
+class TrackAdapter(
+    val trackList: List<Track>
+): RecyclerView.Adapter<TrackAdapter.TrackViewHolder>() {
+
+    class TrackViewHolder(itemView: View): RecyclerView.ViewHolder(itemView) {
+        private val trackName: TextView
+        private val artistName: TextView
+        private val trackTime: TextView
+        private val artworkUrl100: ImageView
+
+        init{
+            trackName = itemView.findViewById(R.id.trackName)
+            artistName = itemView.findViewById(R.id.artistName)
+            trackTime = itemView.findViewById(R.id.trackTime)
+            artworkUrl100 = itemView.findViewById(R.id.imageTrack)
+        }
+
+        fun bind(model: Track) {
+            trackName.text = model.trackName
+            artistName.text = model.artistName
+            trackTime.text = model.trackTime
+            Glide.with(itemView)
+                .load(model.artworkUrl100)
+                .placeholder(R.drawable.placeholder)
+                .fitCenter()
+                .transform(RoundedCorners(10))
```

Comment:
🍏 В RoundCorners нужно передать количество пикселей, которое будет разным в зависимости от экрана. Используй getDimensionPixelSize() вместо хардкода.
Также все трасформации, включая fitCenter, можно указать внутри метода transform. Тогда трансформации будут применены одновременно и не затрут друг друга.

```suggestion
                .transform(
                FitCenter(),
                RoundedCorners(10)
                )
```